### PR TITLE
Re-formatted several more scripts

### PR DESCRIPTION
**Pull request by riej**
oc_anim (little fix), oc_auth, oc_bell, oc_sys, oc_titler

### CODE REVIEW

**Comment by mailani19** [comment]
So what is it that reformatting these is supposed to do?  use up less script time?  make them run a bit faster?  or just look better?

**Comment by riej** [comment]
Just look better.

**Comment by RestlessSwords** [comment]
one problem with reformatting is that it breaks some of the 'diff' programs that show differences between versions.  it depends on how fancy the diff program is. (does it ignore spaces, does it ignore carriage returns, etc.

**Comment by zontreck** [comment]
Can we not get unnecessary spaces added please? Thank you.

**Comment by riej** [comment]
For constants (like message map)?
Can you please show where?

File: src/collar/oc_anim.lsl [hunk 1]
```diff
@@ -52,38 +52,42 @@ integer CMD_SAFEWORD = 510;
 integer EXT_CMD_COLLAR = 499; // Added for collar or cuff commands to put ao to pause or standOff
 integer ATTACHMENT_RESPONSE = 601;
 
-integer NOTIFY = 1002;
+integer NOTIFY  =  1002;
 integer LOADPIN = -1904;
-integer REBOOT = -1000;
+integer REBOOT  = -1000;
+
```
Comment (nirea) [inline src/collar/oc_anim.lsl]: Could we stick to just one space on either side of the equals sign?  While I agree that right-aligning them looks prettier, it's more hassle to maintain unless everyone's editor does it automatically (and I don't know of an LSL editor that will).

File: src/collar/oc_bookmarks.lsl [hunk 2]
```diff
@@ -1,523 +1,606 @@
...
+}
+*/
 
 Dialog(key kRCPT, string sPrompt, list lChoices, list lUtilityButtons, integer iPage, integer iAuth, string sMenuType) {
-    key kMenuID = llGenerateKey();
-    llMessageLinked(LINK_DIALOG, DIALOG, (string)kRCPT + "|" + sPrompt + "|" + (string)iPage + "|" + llDumpList2String(lChoices, "`") + "|" + llDumpList2String(lUtilityButtons, "`") + "|" + (string)iAuth, kMenuID);
-    integer iIndex = llListFindList(g_lMenuIDs, [kRCPT]);
-    if (~iIndex) g_lMenuIDs = llListReplaceList(g_lMenuIDs, [kRCPT, kMenuID, sMenuType], iIndex, iIndex + g_iMenuStride - 1);
-    else g_lMenuIDs += [kRCPT, kMenuID, sMenuType];
+  key kMenuID = llGenerateKey();
```
Comment (zontreck) [inline src/collar/oc_bookmarks.lsl]: It appears you used a search and replace for  a certain number of spaces. LSL infact does not use tabs for indentation, it uses five spaces.

File: src/collar/oc_bookmarks.lsl [hunk 3]
```diff
@@ -1,523 +1,606 @@
...
-        string sLocationx = llList2String(lPieces, iHttploc + 5);
-        string sLocationy = llList2String(lPieces, iHttploc + 6);
-        string sLocationz = llList2String(lPieces, iHttploc + 7);
-        sStr = sStringToKeep + sRegion + "(" + sLocationx + "," + sLocationy + "," + sLocationz + ")";
-        //Debug("Converted slurl, sending for processing..." + sStr);
-        return sStr; //successful conversion, send converted string assembly
-    }
-    //Debug("No slurl detected, sending for processing anyways :" + sStr);
-    return sStr; //failed conversion, send raw string assuming there's no Slurls
+string ConvertSLUrl(string sStr) { // convert the slurl http strings to region (xxx,xxx,xxx)
```
Comment (zontreck) [inline src/collar/oc_bookmarks.lsl]: It appears the indentation in this function was also removed and removed with fewer spaces as well as functions above it.

**Pull request closed without merging**